src/common/packager.ts

```diff
@@ -16,6 +16,7 @@ import * as Q from "q";
 import * as path from "path";
 import * as XDL from "../common/exponent/xdlInterface";
 import * as url from "url";
+import * as vscode from "vscode";
```

Comment:
As above, we can't directly refer to `vscode` in here since it is also accessed from within the separate debug adapter process.